client.go

```diff
@@ -118,3 +118,80 @@ func newClientUsingAddresses(port uint16, remoteAddresses []string) (*Client, er
 		return newClientUsingCluster(cluster)
 	}
 }
+
+// Fetch is a simple invocation of the NewFetchValueCommand
+func (c *Client) Fetch(bucketType, bucket, key string) ([]*Object, error) {
+	cmd, err := NewFetchValueCommandBuilder().
+		WithBucketType(bucketType).
+		WithBucket(bucket).
+		WithKey(key).
+		Build()
+
+	if err != nil {
+		return nil, err
+	}
+	if err = c.cluster.Execute(cmd); err != nil {
+		return nil, err
+	}
+
+	res := cmd.(*FetchValueCommand)
+	if res.Error() != nil || res.Response.IsNotFound {
+		return nil, res.Error()
+	}
+
+	return res.Response.Values, nil
+}
+
+// Store is a simple invocation of the StorehValueCommand
+func (c *Client) Store(bucketType, bucket, key string, data []byte) ([]*Object, error) {
+	obj := &Object{
+		ContentType:     "application/json",
+		Charset:         "utf-8",
+		ContentEncoding: "utf-8",
+		Value:           data,
+	}
```

Comment:
Not sure how you best want to handle letting the user configure this... perhaps delegate the object creation to the calling code, and have this method just accept a *riak.Object instead of an array of bytes?